internal/fourslash/fourslash.go

```diff
@@ -227,7 +227,11 @@ func (f *FourslashTest) nextID() int32 {
 }
 
 func (f *FourslashTest) initialize(t *testing.T, capabilities *lsproto.ClientCapabilities) {
-	params := &lsproto.InitializeParams{}
+	params := &lsproto.InitializeParams{
+		InitializeParamsBase: lsproto.InitializeParamsBase{
+			Locale: ptrTo("en-US"),
```

Comment:
Probably we should still let this be unspecified, but then treat unspecified as `en-US` in the LS.